sdk/src/test/resources/geocode.json

```diff
@@ -0,0 +1 @@
+{"meta":{"code":200},"addresses":[{"latitude":40.7039,"longitude":-73.9867,"geometry":{"type":"Point","coordinates":[-73.9867,40.7039]},"formattedAddress":"20 Jay Street, Brooklyn, New York, NY 11201 USA","country":"United States","countryCode":"US","countryFlag":"🇺🇸","state":"New York","stateCode":"NY","postalCode":"11201","city":"New York","borough":"Brooklyn","county":"Kings County","neighborhood":"DUMBO","number":"20","confidence":"exact"}]}
```

Comment:
I think it's worth using a JSON formatter / pretty printer on this for readability, esp given that it's a one liner

Reply:
Good call. And thanks @jsani-radar for taking the initiative and knocking this out